.github/workflows/trigger-dev-workflow.yaml

```diff
@@ -0,0 +1,77 @@
+name: Trigger Deploy to AWS Dev
+run-name: "Trigger Deploy - Service: ${{ inputs.service }}, Version: ${{ inputs.version }}, Dev ID: dev${{ inputs.dev_id }}, Config Name: ${{ inputs.config_name }}, Terraform Branch: ${{ inputs.tf_branch }}, Event Type: ${{ inputs.event_type }} to AWS Dev by @${{ github.actor }}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      container_tag:
+        type: string
+        description: "The container tag to deploy. Example: `5.0.0`"
+        required: true
+        default: main
```

Comment:
```suggestion
```
I don't think a default makes sense for this one - maaaaybe `latest`, but we aren't currently using that tag very consistently